### ISSUE TITLE
Delete a board on deck app does not work

### ISSUE THREAD

**Issue by pixel8383** [nextcloud/deck#5521]
Any user (admin or not) creates a new board inside the app but when trying to delete it, the process fails and the board doesn't get deleted.

During the process an error message is shown on the browser Console:

` GET https://website.com/core/templates/message.html 404 (Not Found)
`

The issue is present on two Nextcloud installs, one running NC 28.0.2RC1 and the second one running 28.0.1. The Deck version is the latest available on both installs (1.12.1).

**To Reproduce**
1. Go to /apps/deck/
2. Click on Add new Board
3. Create the Board and click on the three dots next to the Board
4. Delete the Board

**Expected behavior**
The Board should be deleted

**Client details:**
 - OS: Windows/Linux
 - Browser: Firefox / Edge / Chrome
 - Device: Linux / Windows

<details>
<summary>Server details</summary>

**Operating system**: Ubuntu

**Web server:** nginx 1.24.0

**Database:**  10.5.21-MariaDB-0+deb11u1 Debian 11

**PHP version:** 8.1

**Nextcloud version:** 28.0.1 and 28.0.2RC1

**Comment by Jerome-Herbinet** [comment]
@pixel8383 

Tested on Nextcloud 28.0.2 + Deck app 1.12.2

It works in this environment, by my side. Also tested on mobile (Deck android app : 1.23.4)

Please update your apps, test again and tell if it works.

Thx

**Comment by pixel8383** [comment]
Hello @Jerome-Herbinet, I am using NC28.0.2 and Deck 1.12.2 but I can't delete the test board I created.

When I click on the three dots next to the board and select "Delete board" what I get is a prompt on the browser reporting " error during model message loading". On the beowser console, the only error I can see is an XHR request to https://mywebsite.com/core/templates/message.html returning a 404 error.

Is there any way to give you a more verbose debug information? I can't see any relevant error on the NC logviewer app.

Thanks for your support.

**Comment by Jerome-Herbinet** [comment]
I suggest that you provide logs (go to logging section of the admin settings), so it will help the developer(s).

Some questions : 
- Does the problem occur with all your boards ? If not, is the board different from others (for example : some special characters in titles, tags, description, or is it shared by you or with you)
- Have you ever tried to delete it from the mobile app (and see if it works)

**Comment by pixel8383** [comment]
I can say the issue is present on any board. I am facing the same error on two different Nextcloud installations (both 28.0.2 with Deck 1.12.2).
I don't get if this error is something related to a misconfiguration of the webserver (but I am using the suggested configurations you can find on the Nextcloud documentation for an nginx webserver) or something else.

As I told you before, no error appears on the nextcloud.log file when I try to delete a board (_should I increase the log level?_).

Anyway, I just tried from the iOS Nextcloud app (never tried before) and I can confirm I can delete the boards correctly.

**Comment by Jerome-Herbinet** [comment]
OK, since you updated from Nextcloud 28.0.2RC1 to 28.0.2 and from Deck 1.12.1 to 1.12.2, did you create new boards ? ... or all your boards are older ? In other words did you made a test or creation then deletion very recently ?

**Comment by pixel8383** [comment]
Yes, I did some new tests today while answering to your questions.

**Comment by pixel8383** [comment]
@Jerome-Herbinet do you have any suggestion for me?

**Comment by Jerome-Herbinet** [comment]
No at this point. I just make sure that more information is provided and that the issue is correctly qualified with tags. Let's wait for developers to take care of it when available.

**Comment by pixel8383** [comment]
I would like to go further on the error I can see on the browser (`GET https://website.com/core/templates/message.html 404 (Not Found)`) when I click on Delete. I have seen the same error on another app so I am supposing this issue could be related to some wrong web server configuration (too strict security maybe?). I would say my configuration is the same as the one [suggested on the Nextcloud documentation](https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html).

Unfortunately I don't know how/what to debug it. Who needs the message.html file? What is that file used for?

**Comment by joshtrichards** [comment]
Can you please check your browser inspector and monitor the *Network* tab while reproducing your situation? I suspect the real error condition is occurring before `/core/templates/message.html` is requested. (Though I don't know why you're getting a 404 on it in <=29).
